.github/workflows/build-test-windows.yml

```diff
@@ -2,11 +2,22 @@
 name: build-and-test-windows
 on:
   push:
-    branches: [ main, master ]
+    branches:
+      - master
```

Comment:
Keep main, it's there for future transition to "main" instead of "master"

Reply:
I missed that in my review. I did see that the other build yaml doesn't have `main` either though, so they'll both need to be updated.